unit_tests/test_install_script.sh

```diff
@@ -272,5 +272,17 @@ testManageClientLibrariesSecurityConfig() {
   assertEquals 0 $?
 }
 
+### Manage client libraries profiling config
+testManageClientLibrariesProfilingConfig() {
+  rm $environment_file
+  echo 'PATH="/usr/local/sbin"' > $environment_file
```

Comment:
Is this really required? I don't understand the relation with what you're testing here

Reply:
Could be, but I'm worried it hides potential problems if the file doesn't exist (which sounds unlikely, but not impossible)

Reply:
I can add something like
```
    if [ ! -f "$etc_environment" ]; then
      $sudo_cmd touch "$etc_environment"
    fi
```
to make sure the file is created if it doesn't exist. `manage_client_libraries_security_config()` function is also vulnerable to the file not existing, then it'd need to be touched up too. @robertpi as you wrote `manage_client_libraries_security_config()` do you have an opinion on behavior when `/etc/environment` doesn't exist?

Reply:
LGTM for this part, thanks!